project_event/models/project_task.py

```diff
@@ -1012,3 +1027,16 @@ def get_confirmation_wizard(self, action):
     def check_task_state(task_state_in):
         return task_state_in in \
             ['draft', 'option', 'postponed', 'canceled']
+
+    def get_booked_attendees(self, attendee):
+        vals = {
+            'start': self.date_start,
+            'stop': self.date_end,
+            'partner_ids': [(6, 0, [attendee])]}
+        v_event = self.env['calendar.event'].new(vals)
+        [(6, 0, self.get_partners())]
+        try:
+            v_event._check_resources_booked(v_event, v_event.partner_ids)
+        except Exception as e:
+            return [e.name, e.name.split(' ')[1], e.name.split(' ')[2]]
```

Comment:
this traceback is displayed when clicking on option:
Traceback (most recent call last):
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 651, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 693, in dispatch
    result = self._call_function(**self.params)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 342, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 335, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 937, in __call__
    return self.method(*args, **kw)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/addons/web/controllers/main.py", line 938, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/addons/web/controllers/main.py", line 926, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/home/rbendhaou/Documents/Projets/Udem/git/odoo/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/rbendhaou/Documents/Projets/Udem/git/project-addons/project_event/models/project_project.py", line 111, in action_option
    return self.get_confirmation_wizard('option')
  File "/home/rbendhaou/Documents/Projets/Udem/git/project-addons/project_event/models/project_project.py", line 207, in get_confirmation_wizard
    res += activity.get_booked_resources()
  File "/home/rbendhaou/Documents/Projets/Udem/git/project-addons/project_event/models/project_task.py", line 724, in get_booked_resources
    hres = child.get_booked_attendees(attendee)
  File "/home/rbendhaou/Documents/Projets/Udem/git/project-addons/project_event/models/project_task.py", line 1041, in get_booked_attendees
    return [e.name, e.name.split(' ')[1], e.name.split(' ')[2]]
AttributeError: 'DataError' object has no attribute 'name'